sycl/test-e2e/OneapiDeviceSelector/discard_filters.cpp

```diff
@@ -0,0 +1,6 @@
+// REQUIRES: gpu
+// RUN: %clangxx -fsycl -fsycl-targets=%{sycl_triple} %S/Inputs/trivial.cpp -o %t.out
+
+// Test discard filters in ONEAPI_DEVICE_SELECTOR.
+// RUN: env ONEAPI_DEVICE_SELECTOR="!*:gpu" %t.out | FileCheck %s --allow-empty
```

Comment:
Needs something like {run-unfiltered-devices}, don’t remember the exact spelling